fossevents/base/mail.py

```diff
@@ -0,0 +1,43 @@
+# Third Party Stuff
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.mail import EmailMultiAlternatives
+from django.template import RequestContext, TemplateDoesNotExist
+from django.template.loader import render_to_string
+
+
+def send_email(subject_template_name, email_template_name, to_emails,
+               context=None, html_email_template_name=None, request=None):
+    ctx_dict = {}
+    current_site = Site.objects.get_current()
+    if request is not None:
+        ctx_dict = RequestContext(request, ctx_dict)
+    # update ctx_dict after RequestContext is created
+    # because template context processors
+    # can overwrite some of the values like user
+    # if django.contrib.auth.context_processors.auth is used
+    if context:
+        ctx_dict.update(context)
+    ctx_dict.update({
+        'site': current_site,
+        'scheme': settings.SITE_SCHEME
+    })
+    subject = (render_to_string(subject_template_name, ctx_dict))
```

Comment:
extra brackets and may be we can remove `subject = ''.join(subject.splitlines())`